Updated documentation with Utility Functions (#229)
ansys/pymotorcad: doc/source/methods/index.rst

defining and modifying Adaptive Templates geometries in Python.
For descriptions of the functions, see :ref:`ref_geometry_shapes`.

Utility functions
------------------

> A number of utility functions are available to the user.
> For descriptions, see

Reviewer: ```suggestion
A number of utility functions are available.
For more information, see
```
Avoid "user" but use "you" instead--unless you are distinguishing between user types  (administrators versus users).

:ref:`ref_utility_functions`.

> Motor-CAD Errors

Reviewer: ```suggestion
Motor-CAD errors
```

----------------

> A class/exception type used for obtaining and handling errors from Motor-CAD.
> For descriptions, see
> :ref:`MotorCAD_errors`.

Reviewer: ```suggestion
A class/exception type is used to obtain and handle errors from Motor-CAD.
For more information, see
:ref:`MotorCAD_errors`.
```
The first line must be a complete sentence, not a fragment.
